node_test.go

```diff
@@ -14,7 +14,7 @@ var _ = Describe("Tests", func() {
 	)
 
 	BeforeEach(func() {
-		handle, err = SetupHandle("qdb://127.0.0.1:30083", 120*time.Second)
+		handle, err = SetupHandle("qdb://127.0.0.1:2836", 120*time.Second)
```

Comment:
Why don't you use the above defined URIs?

Reply:
Done.